internal/appconfig/from_machine_set.go

```diff
@@ -108,10 +108,10 @@ func fromAppAndOneMachine(appCompact *api.AppCompact, m machine.LeasableMachine,
 		})
 	}
 	if len(m.Machine().Config.Mounts) > 0 {
-		mounts = &Volume{
-			Destination: m.Machine().Config.Mounts[0].Path,
-		}
+		mount := m.Machine().Config.Mounts[0]
+		mounts = append(mounts, Mount{Source: mount.Name, Destination: mount.Path})
```

Comment:
this is a total nit, but imo setting the slice to a new value seems more clear than appending to a nil slice
gave me the impression that the machine _could_ end up with multiple mounts, even if in reality it couldn't
```suggestion
		mounts = []Mount{{Source: mount.Name, Destination: mount.Path}}
```